### ISSUE TITLE
[Dashboard] Issue in Downloading Proper Reports Directory from Dashboard

### ISSUE THREAD

**Issue by malshaniS** [wso2/testgrid#963]
**Description:**

Only the automation.log file can be downloaded from the results section of the TestGrid dashboard for the "wso2ei-intg" build. But we move the entire execution result directory from the remote directory. Because there are multiple profiles in EI, the result directory contains multiple subdirectories with each contains the surefire report and automation.log file of their related test suite.

<img width="249" alt="screen shot 2018-08-23 at 1 06 07 pm" src="https://user-images.githubusercontent.com/40388545/44511535-87b79380-a6d5-11e8-8513-1e68b0da7e93.png">

Downloaded integration-test.zip contains only the automation.log file.

**Suggested Labels:**
Bug

**Suggested Assignees:**

**Affected Product Version:**

**OS, DB, other environment details and versions:**    

**Steps to reproduce:**

Go to "https://testgrid-live-dev.private.wso2.com/wso2ei-intg" and download the results from scenario execution summary.

**Related Issues:**

**Comment by kasunbg** [comment]
@malshaniS @azinneera   where are we on this issue? Do we still have this issue?

**Comment by malshaniS** [comment]
No. This issue has been fixed. We can download the entire results directory with multiple surefire reports now.
